build/index.js

```diff
@@ -175,6 +176,9 @@ async function buildDocument(document) {
   // Extract all the <h2> tags as they appear into an array.
   doc.toc = extractTOC($);
 
+  // Check and scrutinize any local image references
+  const fileAttachments = checkImageReferences(doc, $, options, document);
```

Comment:
Unlike the `injectFlaws` this function does more stuff. It also returns all the absolute paths to the file attachments that need to be copied into `client/build/`. So, more reason why this monster function doesn't need to be nestled into the `build/flaws.js` file.